pkg/interact/telegram.go

```diff
@@ -62,7 +68,14 @@ type TelegramReply struct {
 }
 
 func (r *TelegramReply) Send(message string) {
-	checkSendErr(r.bot.Send(r.session.Chat, message))
+	var left, right int
+	for left, right = 0, maxMessageSize; right < len(message); left, right = right, right+maxMessageSize {
```

Comment:
you only need to split the message when its length > maxMessageSize?

Reply:
yes